.drone.yml

```diff
@@ -8,6 +8,7 @@ pipeline:
     commands:
       - go get -u github.com/golang/dep/cmd/dep
       - dep ensure
+      - go test ./...
```

Comment:
For other plugins we are using something like that to avoid testing of vendor directory:

```
  test:
    image: golang:1.9
    pull: true
    commands:
      - go vet
      - |
        for PKG in $(go list ./... | grep -v /vendor/); do
          go test -cover -coverprofile $GOPATH/src/$PKG/coverage.out $PKG
        done
```

Reply:
@tboerger as of go 1.9 they ignore the vendor directory for tests :)

Reply:
I went ahead and pushed a commit to align build stage testing with other plugins.